packages/@romefrontend/ast/css/core/CSSAtRule.ts

```diff
@@ -0,0 +1,21 @@
+import {CSSBlock, NodeBaseWithComments} from "@romefrontend/ast";
+import {createBuilder} from "@romefrontend/ast/utils";
+import {AnyCSSValue} from "../../../css-parser/types";
+
+export type CSSAtRule = NodeBaseWithComments & {
+	type: "CSSAtRule";
+	name: string;
+	prelude: Array<AnyCSSValue>;
+	block: CSSBlock | null;
```

Comment:
We actually don't use `null` anywhere in Rome, except when checking values given to use by Node or native methods.

```suggestion
	block?: CSSBlock;
```